src/test/test_dash.h

```diff
@@ -143,7 +143,7 @@ CBlock getBlock13b8a();
 // BOOST_CHECK_EXCEPTION predicates to check the specific validation error
 class HasReason {
 public:
-    explicit HasReason(const std::string& reason) : m_reason(reason) {}
+    HasReason(const std::string& reason) : m_reason(reason) {}
```

Comment:
need to drop this removal from 6de74e958db447f641adcdcddc7871dbc57c39b2

Reply:
done